litmus-portal/frontend/src/components/UserManagementTab/CreateUser/index.tsx

```diff
@@ -0,0 +1,234 @@
+import {
+  Button,
+  FormControl,
+  IconButton,
+  Input,
+  InputAdornment,
+  InputLabel,
+} from '@material-ui/core';
+import Divider from '@material-ui/core/Divider';
```

Comment:
I think you can call for these imports(TextField, Typography, etc others from MUI) together at the top like you did on line 1-8.